client/src/client.ts

```diff
@@ -1146,7 +1146,7 @@ export class LanguageClient {
 		this.refreshTrace(connection, false);
 		let initOption = this._clientOptions.initializationOptions;
 		let initParams: InitializeParams = {
-			processId: process.pid,
+			processId: typeof process !== 'undefined' ? process.pid : 0,
```

Comment:
The LSP spec says `null` is OK, but the TypeScript type definition forbids `null`. I will leave it as `0` for now so it compiles here.